lonestar/libdistbench/include/DistBench/Start.h

```diff
@@ -117,6 +117,15 @@ static void marshalGPUGraph(
 }
 #endif
 
+template <typename NodeData, typename EdgeData>
+using DistGraphPtr =
+    std::unique_ptr<galois::graphs::DistGraph<NodeData, EdgeData>>;
+template <typename Graph>
+using SubstratePtr = std::unique_ptr<galois::graphs::GluonSubstrate<Graph>>;
```

Comment:
This seems to be unused; if so, remove it.

Note that not any graph will work with GluonSubstrate (there's a set of functions GluonSubstrate will look for), so this probably won't even work depending on what Graph is defined as.

Reply:
A debug relic lol but thanks for pointing this out :p